### ISSUE TITLE
Get-WGUpgrade incorrectly parsing

### ISSUE THREAD

**Issue by jdhitsolutions** [jdhitsolutions/WingetTools#4]
Regex pattern in Get-WGUpgrade is failing.

**Comment by jdhitsolutions** [comment]
```powershell
PS C:\> Get-WGUpgrade

Name                                                                     ID
----                                                                     --     
Visual Studio Community 2022 Microsoft.VisualStudio.2022.Community       17.1.0 
Discord                      Discord.Discord                             1.0.90…
MozBackup                    JasnaPaka.MozBackup                         Unknown
Mozilla Thunderbird          Mozilla.Thunderbird                         91.6.1 
WinRAR                       RARLab.WinRAR                               6.10.0 
Zoom                         Zoom.Zoom                                   5.9.37…
```

**Comment by jdhitsolutions** [comment]
Fixed in v1.2.0. Same problem as Issue#3
